app/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,29 @@
+import androidx.compose.desktop.ui.tooling.preview.Preview
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.window.Window
+import androidx.compose.ui.window.WindowPosition
+import androidx.compose.ui.window.application
+import androidx.compose.ui.window.rememberWindowState
+import java.awt.Dimension
+import view.utils.SelectInitDialogWindow
+
+@Composable
+@Preview
+fun App() {
```

Comment:
Can you please explain why it will?

Reply:
This is a public top-level function so it is visible and can be called absolutely anywhere in the code. For the reader of this file it gives an impression that the function is used somewhere outside this file even though it is not. For the developer, if anywhere in your project you type `A` IDE's auto-completion will suggest to use this `App`.

In general, you should use the most strict visibility modifier you can to avoid all these problems. `private` here since the function is only used inside this file.

Reply:
Oh, i see) My bad))))